recipe/meta.yaml

```diff
@@ -1,88 +1,108 @@
-{% set version = "4.2.2" %}
+{% set name = "ffmpeg" %}
+{% set version = "6.1.1" %}
+# win specific vars
+{% set version_majmin = '.'.join(version.split('.', 2)[:2]) %}
+{% set release_date = "2023-12-31" %}
+{% set release_time = "12-55" %}
 
 package:
-  name: ffmpeg
+  name: {{ name }}
   version: {{ version }}
 
 source:
-  - url: https://ffmpeg.org/releases/ffmpeg-{{ version }}.tar.gz  # [not win]
-    sha256: 83f9a9aa0acf8036daf47494d99a8c31154a18ebb6841d89878ba47783559bd0  # [not win]
-
-  - url: https://ffmpeg.zeranoe.com/builds/win64/shared/ffmpeg-{{ version }}-win64-shared.zip  # [win]
-    sha256: 5aedf268952b7d9f6541dbfcb47cd86a7e7881a3b7ba482fd3bc4ca33bda7bf5  # [win]
-
-  - url: https://ffmpeg.zeranoe.com/builds/win64/dev/ffmpeg-{{ version }}-win64-dev.zip  # [win]
-    sha256: f4885f859c5b0d6663c2a0a4c1cf035b1c60b146402790b796bd3ad84f4f3ca2  # [win]
+  url: https://github.com/FFmpeg/FFmpeg/archive/refs/tags/n{{ version }}.tar.gz
+  sha256: 7c1ebea95d815e49c1e60c7ee816410dec73a81b8ac002b276780d2f9048e598
+  # This patch can be removed once the fix below is merged and uploaded,
+  # the libxml2 2.13.1 buildout is done, and the libxml2 aggregate pinning
+  # has been updated.
+  # https://github.com/AnacondaRecipes/libxml2-feedstock/pull/16
+  patches:
+    - patches/0001-use-include-path-of-our-libxml2.patch # [win]
```

Comment:
We might need https://github.com/conda-forge/ffmpeg-feedstock/blob/main/recipe/patches/pkgconfig_generate_windows_llvm.patch.

Reply:
included